proposals/working-group/Comparisons.adoc

```diff
@@ -0,0 +1,36 @@
+= Pros/Cons of Each Proposal
+
+== MPWG
+Pros
+
+* Flatter governing structure as steering committee, marketing committee and specification committees share the same membership.
+* Addresses IP gap in existing MP development
+* Provides complete autonomy of how MP evolves as governing structure is not part of a non-MP specific working group
+* Provides a lower barrier to entry in terms of both cost and process for the MP technology
+* Continues a model that has produced multiple independent implementations of novel technologies over a number of years
+* Allows for large companies to sponsor specification development at a zero cost entry point
+* Less restrictive use of compatibility and logo usage
+
+Cons
+
+* Duplicate working group participation agreements need to be signed
+* My have duplicate fees depending on the membership level if both MPWG and CN4J/Jakarta are joined
+* Tends to not commit to longer term backwards compatibility 
```

Comment:
Another potential con...
* Duplicate working group participation across common organizations.   (There are at least 9 organizations currently involved with both MP and Jakarta.  These same organizations will need to participate in both WG organizations.)

I guess this could also be listed as a Pro for CN4J WG.  Less required participation at the WG level if there is only one WG governing both projects.

Reply:
Yes the biggest issue with 2 working groups is the complete pointless duplication of effort in bureacracy

Reply:
It's a matter of perspective, I suppose. Red Hat and Payara are approaching this differently. Correct me if I am incorrect, but I suspect that Payara sees MicroProfile as an extension of Jakarta EE. Red Hat sees it as something separate from Jakarta EE. They can (and do) both leverage from each other when it makes sense, but some MicroProfile implementations never intend to implement Jakarta EE and therefore would like the autonomy to make decisions that are better for MicroProfile. Autonomy = independent steering committee in our (Red Hat's) mind.

Reply:
No you are incorrect. Payara sees the proposal of a separate working group as pointless bureaucracy. We may implement a MicroProfile only implementation but will still see a separate WG as pointless bureaucracy and a needless drain on our resources.